### PR TITLE
Added `saddle_holder` for bicycle_parking quest

### PR DESCRIPTION
**Pull request by mcliquid**
The bike park season is almost over, but I think it's still relevant :)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -768,6 +768,7 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_bicycle_parking_type_locker">Locker</string>
     <string name="quest_bicycle_parking_type_building">Building</string>
     <string name="quest_bicycle_parking_type_handlebarholder">Handlebar holder</string>
+    <string name="quest_bicycle_parking_type_saddleholder">Handlebar holder</string>
```
Comment (mnalis) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="quest_bicycle_parking_type_saddleholder">Saddle holder</string>
```

**Review by mnalis — COMMENTED** [review]
Nice, although a different picture would be better if available. Green-black saddle is fancy, but does not show well on green-black background, especially on a tiny mobile phone picture

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (mcliquid) [reply app/src/main/res/values/strings.xml]: Thanks!

**Comment by mcliquid** [comment]
> Nice, although a different picture would be better if available.

@mnalis  Do you have a suggestion?

**Comment by mnalis** [comment]
> @mnalis Do you have a suggestion?

- Well, if you have seen one in your area, obviously you could take pictures (from several angles) and use that.
- I usually search on https://commons.wikimedia.org (although having a more precise search term than _"saddle holder"_ would help. Perhaps German name for those holders?) or https://www.geograph.org.uk/
- failing that, one might use https://duckduckgo.com/ image search with license filter set to Creative Commons, or https://search.creativecommons.org/
- another alternative (if  you know/can guess where such parkings might could be found) might be to use [Panoramax, KartaView, OpenTrailView, Mapillary or Mapilio](https://wiki.openstreetmap.org/wiki/Street-level_imagery_services) "street view" - their images should be CC-BY-SA licensed, which is fine for us, provided they're of sufficient quality.

**Comment by mnalis** [comment]
this one for example seems to be cc-by-sa 2.0: https://flickr.com/photos/phonescoop/2287609845/

**Comment by mcliquid** [comment]
@mnalis The latest images from my camera roll you can find here: https://commons.wikimedia.org/w/index.php?title=Special:ListFiles/Mcliquid&ilshowall=1

**Comment by mnalis** [comment]
that https://commons.wikimedia.org/wiki/File:Fahrradparkplatz_Sattelhalter_1.jpg looks like it would be discernible on small screens the best IMHO

**Comment by mcliquid** [comment]
Perfect match! Changed it already here: https://github.com/Helium314/SCEE/pull/652/commits/3aaa1a4d5f383b51ad04558a0942890f3c5c1111

**Review by mnalis — APPROVED** [review]
Assuming that you tested that it works, this looks good to me!

**Comment by Helium314** [comment]
Looks fine, thanks!

**Comment by Helium314** [comment]
Oh, I noticed you added the string to SC strings, not SCEE. No real problem, but it won't be translatable in Weblate.

**Comment by mcliquid** [comment]
Thanks! I will update them.

**Pull request merged**